libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -79,19 +79,13 @@ void AP_RangeFinder_MAVLink::update(void)
     if (AP_HAL::millis() - state.last_reading_ms > AP_RANGEFINDER_MAVLINK_TIMEOUT_MS) {
         set_status(RangeFinder::Status::NoData);
         state.distance_m = 0.0f;
+        state.signal_quality_pct = state.signal_quality_pct == RangeFinder::SIGNAL_QUALITY_NODATA ?
+                RangeFinder::SIGNAL_QUALITY_NODATA : RangeFinder::SIGNAL_QUALITY_MIN;
```

Comment:
I would set signal_quality_pct to SIGNAL_QUALITY_NODATA in this case. Setting to MIN implies that we have a measurement, we know the quality, and the quality is terrible. Whereas NODATA means we don't have a measurement, or we have a measurement and we don't know the quality.

Reply:
Yes. Good point. Changed.